docs/testing/README.md

```diff
@@ -0,0 +1,26 @@
+# Testing in LocalStack
+
+- [Integration Tests](integration-tests/README.md)
+- [Parity Testing](parity-testing/README.md)
+- [Multi-account and Multi-region Testing](multi-account-region-testing/README.md)
+- [Terraform Tests](terraform-tests/README.md)
+
+## Rules for stable tests
+
+Through experience, we encountered some guiding principles and rules when it comes to testing LocalStack.
+These aim to ensure a stable pipeline and keep flakes minimal. Any newly added test and feature should keep these in mind!
+
+| **ID** | **Rule** |
+|-------|-----------|
+| R01   | Inform code owners and/or test authors when a test is marked as flaky so that it can be fixed ASAP. |
+| R02   | Do not assume external dependencies (AWS resources, files, packages, images, licenses) are indefinitely available. |
+| R02a  | Where possible, add automatic creation of dependencies if they no longer exist (e.g., S3 buckets, roles). |
+| R02b  | If on-demand creation is not possible, make access failures clearly visible for further investigation. |
```

Comment:
does this just refer to error reporting?

Reply:
Yes, access errors should be clearly marked as such, and as Alex mentioned should lead to fast failures.
This helps avoiding needless debugging of our own code when the issue lies outside of it.

In the cases where this was relevant in the past, we actually did well here. But making it a general rule will help us keep it like this in the future. If this is too trivial, we can also remove it again, though :)

Reply:
Still a bit confused about the wording around `access` here. Since we're for the most part in the context of AWS this feels like you're talking about IAM permissions. Maybe rewording this might clear things up? E.g. retrieving/using vs. accessing?